app/build.gradle

```diff
@@ -8,8 +9,8 @@ android {
     defaultConfig {
         minSdkVersion 15
         targetSdkVersion 27
-        versionCode 1
-        versionName "1.0"
+        versionCode 2
+        versionName "1.1"
```

Comment:
version name and version code are not needed to be updated

Reply:
reverted